### DISCUSSION TITLE
Webhook. What is the recommended flow to deliver results

### DISCUSSION THREAD

**Discussion by timrvv** [impress-org/givewp#7066] [Developer Q&A]
Our gateway partner uses webhooks to deliver transaction result. What is the recommended path for this workflow? I've looked up [webhook documentation](https://givewp.com/documentation/add-ons/webhooks/), but it's still not clear what is the best path. Which command should we use in createPayment? How donation will be updated based on the transaction result?

**Comment by jonwaldstein** [comment]
Hey @timrvv,

Our webhooks add-on is for _sending_ events - not receiving.  Are you interested in building a custom gateway?
